pkg/cmd/scan.go

```diff
@@ -216,6 +216,9 @@ func scanRun(opts *pkg.ScanOptions) error {
 
 	alerter := alerter.NewAlerter()
 
+	// For now, we only use the global printer to print progress and information about the current scan, so we simply use stdout by default.
+	globaloutput.ChangePrinter(output.GetPrinter(output.OutputConfig{Key: output.ConsoleOutputType}, opts.Quiet))
```

Comment:
You would have to iterate on the real resources to see if that conflict. If one of the output is json or plan in stdout we don't want to show the progesssbar as it will break the output.

Also remember that this is not only for progressbar we have some other print that use globalprinter and should not be displayed in these cases

Reply:
Yeah I should've think about that, just updated the changes